README.md

```diff
@@ -30,32 +30,48 @@ Edit this document to include your answers after each question. Make sure to lea
 
 1. Describe the biggest difference between `.forEach` & `.map`.
 
+    The forEach method cycles through an array applying a function to each element. while .map creates a new array after it cycles through applying its function to each element.
+
+
 2. What is the difference between a function and a method?
 
+    Method is the blue print to carry out the heist. the functions are the bank robbers completing there assigned task
+
+
```

Comment:
Great explainer, its good to remember the main difference is that methods are functions within an object.